blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -327,7 +327,8 @@ SQL Standard and PostgreSQL compatibility improvements
   scalar function that yields the current value of the setting.
 
 - Allow :ref:`sql_administration_udf` to be registered against the
-  ``pg_catalog`` schema.
+  ``pg_catalog`` schema. This also extends :ref:`scalar_current_schema` to be
```

Comment:
This is unrelated to the release, can we at least put it into a dedicated commit?

Reply:
yep, will split the commit

Reply:
it wasn't completely unrelated, moved it up from https://github.com/crate/crate/pull/8849/commits/f3aea20b6e801f66948a0e454720bdda5e7b3700#diff-8918b36d12905ff85de79bcbdcf052aaL448

Reply:
hmm shall i revert commit split then?